cmd/tailscale/cli/debug.go

```diff
@@ -789,3 +802,102 @@ func runCapture(ctx context.Context, args []string) error {
 	_, err = io.Copy(f, stream)
 	return err
 }
+
+var debugPortmapArgs struct {
+	duration time.Duration
+	ty       string
+}
+
+func debugPortmap(ctx context.Context, args []string) error {
+	ctx, cancel := context.WithTimeout(ctx, debugPortmapArgs.duration)
+	defer cancel()
+
+	portmapper.VerboseLogs = true
+	switch debugPortmapArgs.ty {
```

Comment:
This is the only substantive change to this function: using a command-line argument instead of an envknob.